web/src/styles/_sass-files/variables.scss

```diff
@@ -5,6 +5,13 @@ $sm: 40em; //tablet + up (640px)
 $md: 50em; //big tablet (800px)
 $lg: 64em; //desktop and up (1024px)
 
+$drop-shadow-xs: 0px, 4px, 8px, 0.1;
+$drop-shadow-xs-darker: 4px, 4px, 8px, 0.25;
```

Comment:
[dust] So it kinda seems like Alph here is its own attribute,

so you could do have a second param to the mixin and have like 0.1 and 0.25 as dark and darker respectively as passed as such 

(I'm not saying that this is better or that anything has to change, just kinda thinking out loud)

Reply:
This variable is a list with 4 items. I think it would be more straightforward to define all the attributes within the list than passing in a value like dark/ darker and having the value or those variables defined in the function. Side note - I'm also planning to speak to design about using USDWS shadows so the darker might go away.